### DISCUSSION TITLE
the nightly build of windows shows many warnings

### DISCUSSION THREAD

**Discussion by l1t1** [duckdb/duckdb#8260] [General]
https://github.com/duckdb/duckdb/actions/runs/5559450871

```
Windows Extensions (64-bit): build/release/deps/projects/Stamp/ZLIB/verify-ZLIB.cmake#L15
File will not be verified since no URL_HASH specified
…
```

**Comment by l1t1** [comment]
the linux build has many too.

https://github.com/duckdb/duckdb/actions/runs/5559450879

**Comment by carlopi** [comment]
Thanks for raising this, I took a look at the warnings (and filed a trivial PR), and while they are mostly harmless they should be eventually dealt with, since they can hide more relevant warnings / generate noise.
